students/aravichander/lesson06/Mailroom3.py

```diff
@@ -1,19 +1,21 @@
-#Mailroom #2
+#Mailroom #3&4
 
 import sys
 import json
 import unittest 
 
-Donors = {"Bill Gates":[10000,12000],
-		  "Jeff Bezos":[50],`
-		  "Mark Zuckerberg": [500,600,700],
-		  "Paul Allen": [250,350,450],
-		  "King of Siam": [200,250]
-		  }
+Donors = {}
+def createlist():
+	Donors["Bill Gates"] = [10000,12000]
+	Donors["Jeff Bezos"]=[50]
+	Donors["Mark Zuckerberg"] = [500,600,700]
+	Donors["Paul Allen"] = [250,350,450]
+	Donors["King of Siam"] = [200,250]
+	
 
 def printdonors():
 	for x,v in Donors.items():
-		print(x,"donated $",v)
+		print("\n{} donated ${}".format(x,v))
```

Comment:
this will all be easeir to test if the functions that create text return a string -- then you can test that (and potentially use it in other ways) -- when you want to print, you can keep that isolated:

`print(donor_listing)` or something like that.